blocktopmenu.php

```diff
@@ -1358,4 +1371,104 @@ public function renderList()
 
         return $helper->generateList($links, $fields_list);
     }
+
+    private function createCategoryNode(array $category, $addThumbnails = false)
```

Comment:
please make createCategoryNode public (see #40). It's very useful to be able to reuse such a helper.